src/gobupload/storage/handler.py

```diff
@@ -146,7 +146,7 @@ def create_temporary_table(self, data):
         fields = [collection['entity_id']]
         # If the collection has state, take begin_geldigheid into account
```

Comment:
We should replace 'begin_geldigheid' in the comment as well

Reply:
Sorry, didn’t change this too much because it will change anyway when updating the query. It was just to get it working